benchmarks/build.xml

```diff
@@ -32,8 +32,8 @@
 	<target name="clean">
 		<delete includeemptydirs="true">
 			<fileset
-				dir="."
-				includes="*.sql,benchmarks-actual.properties, liferay/**, ${build.dir}/**"
+				dir="${basedir}"
+				includes="benchmarks-actual.properties, liferay/**, ${build.dir}/**"
```

Comment:
@slnn The commit is "Do cleanup before each invoke" but actually you also changed the basedir and removed an entry from the fileset. Please split commits to make it clear, or drop the changes if they are not actually needed.